build/ci.go

```diff
@@ -158,6 +158,13 @@ func doInstall(cmdline []string) {
 		build.MustRun(goinstall)
 		return
 	}
+	// If we are cross compiling to ARMv5 ARMv6 or ARMv7, clean any prvious builds
+	if *arch == "arm" {
+		os.RemoveAll(filepath.Join(runtime.GOROOT(), "pkg", runtime.GOOS+"_arm"))
+		for _, path := range filepath.SplitList(build.GOPATH()) {
+			os.RemoveAll(filepath.Join(path, "pkg", runtime.GOOS+"_arm"))
+		}
+	}
```

Comment:
Ah that's doable, but then building each of the tools from `cmd` will rebuild ll of go-ethereum, all our deps as well as the entire stdlib. That will blow up build times a lot.